silk/unit_tests/silk_run_test.py

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from src.tests import silk_run
+from silk.tests import silk_run
 import os
 import datetime
 
-RESULT_LOG_PATH = '/opt/openthread_test/results/'+'silk_run_0502_A/'
+RESULT_LOG_PATH = '/opt/openthread_test/results/'+'silk_run/'
```

Comment:
Suggest to append the current date to 'silk_run_' so all tests in the same day will be in one folder for easy access.

Reply:
will fix it